src/platform/site-wide/wc-loader.js

```diff
@@ -0,0 +1,6 @@
+import 'web-components/dist/component-library/component-library.css';
+import { applyPolyfills, defineCustomElements } from 'web-components/loader';
+
+applyPolyfills().then(() => {
+  defineCustomElements();
+});
```

Comment:
This is how we apply polyfills (for IE 11) and get the browser to understand what to do when it finds a Web Component in the html.

View the relevant Stencil docs here: https://stenciljs.com/docs/react